### PR TITLE
the react example made another RealmTasks implementation

### PR DESCRIPTION
**Pull request by gritzko**
Pending issues:
* magic IP/URI discovery
* React Native 0.40 has some antics: item list starts at a negative Y (maybe need a RN expert here, or just wait for a new RN version)

### CODE REVIEW

File: examples/ReactExample/components/login.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import Realm from 'realm';
+import styles from './styles';
+import TodoApp from './todo-app';
+
+
+export default class LoginScreen extends React.Component {
+
+    constructor (props) {
+        super(props);
+        this.state = {login: 'user', password: '...'};
```
Comment (kristiandupont) [inline examples/ReactExample/components/login.js]: Is the "..." password a placeholder? We should probably use the `placeholder` attribute of the `TextInput` element instead.

File: examples/ReactExample/components/login.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                    password: value.password||'password'
+                });
+            }
+        } ).done();
+    }
+
+    submit () {
+        console.log('LOGIN')
+        Realm.Sync.User.login(
+            "http://192.168.157.64:9080/",
```
Comment (kristiandupont) [inline examples/ReactExample/components/login.js]: I suppose we should extract this to somewhere. Where is it stored in the other examples?

File: examples/ReactExample/components/login.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            this.state.password,
+            (error, user) => {
+                if (error) {
+                    this.state.error = error;
+                    this.forceUpdate();
+                } else {
+                    delete this.state.error;
+                    AsyncStorage.setItem('creds', JSON.stringify(this.state));
+
+                    console.log('BINGO');
```
Comment (kristiandupont) [inline examples/ReactExample/components/login.js]: We should probably remove these before merging

File: examples/ReactExample/components/realm.js [hunk 4]
```diff
@@ -20,23 +20,45 @@
...
+    name: 'TaskListList',
+    properties: {
+        id: 'int',
+        items: {type: 'list', objectType: 'TaskList'},
     },
+    primaryKey: 'id'
 };
 
-export default new Realm({schema: [Todo, TodoList]});
+var adminToken = "ewoJImlkZW50aXR5IjogImFkbWluIiwKCSJhY2Nlc3MiOiBbInVwbG9hZCIsICJkb3dubG9hZCIsICJtYW5hZ2UiXQp9Cg==:I1mEfddNdQo4wS/7dVgEIEm9pv7pLEyyx3zmGgWMCxkEHOxz8DxLjm/yLD8OD7iE9XcCHy4xs6SOu2ZbbRysezssyY+3z/anrz7u/EBDvyfHPRakaw3rTpD6RcTb4SaAeLBagtDP7YqycneZrGd3oUXofvJdWg4YpDOvr/+1zwntt3DGg5D1ghKzSWcF7FnosswjiqGk91t2hbZbPYNEPEJKwkCPj0exOZqQ73627rD9Jzbr3CcHpp0V4U1BEhVfgZyo9dchWYT9Qw0rkqjqc5ylvR73Ihpa9hN9+wKxhU1K6nNzsbhrD0sMfOOlxsuduWHrhiT8vkTL/WFOO4vC1Q==";
```
Comment (kristiandupont) [inline examples/ReactExample/components/realm.js]: How is this stored in the other examples?

File: examples/ReactExample/components/login.js (hunk 2 above)
Comment (gritzko) [inline examples/ReactExample/components/login.js]: There is some magic; they take ip from ifconfig and substitute it there.
That potentially given some weird errors if your WiFi address changes.
There must be a better solution.

File: examples/ReactExample/components/login.js (hunk 1 above)
Comment (gritzko) [inline examples/ReactExample/components/login.js]: Thanks.

File: examples/ReactExample/components/realm.js (hunk 4 above)
Comment (gritzko) [inline examples/ReactExample/components/realm.js]: Oops, not needed anymore.

File: examples/ReactExample/components/login.js (hunk 2 above)
Comment (gritzko) [inline examples/ReactExample/components/login.js]: I think, I'll put all such things into a separate config file.

**Pull request closed without merging**